[1~3주차] layer_normalization & adam_optimizer & lstm & dcgan / 박지은

Jieun-Park99 opened the pull request:
# What does this PR do?
Added Korean files for week 1 ~ week 3 in the `docs/ko/` 
- Change docs folder file which written in English to Korean.
- Translate `normalization / layer_norm / index.html` for week 1
- Translate `optimizers / adam_warmup_cosine_decay.html` for week 1.
- Translate `lstm / index.html` for week 2.
- Translate `gan / dcgan / index.html` for week 3.

# Before submitting
- [x]  This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

- [x] Did you make sure to update the documentation with your changes? Here are the
[documentation guidelines](https://github.com/labmlai/annotated_deep_learning_paper_implementations/tree/master/docs), and
[here are tips on formatting html and css](https://chanrankim.notion.site/2c2967ff842344a78a04f50d8274c802?pvs=4)

- [ ] Did you write any new necessary tests?

# Who can review?
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@Bandi120424 , @Jieun-Park99 , @kimdahyeon977 , @minyoungci , @mlleo , @movie5, @nehcream2od, @rimiyeyo @russellgeum @Youngmi-Park 

Pseudo Lab, Please review this PR.

Jieun-Park99 committed:
✨ feat normalization / layer_norm / index.html
✨ feat optimizers / adam_warmup_cosine_decay.html

Jieun-Park99 committed:
✨ feat lstm / index.html

Jieun-Park99 committed:
✨ feat  gan / dcgan / index.html

movie5 commented:
[스타일 가이드라인](https://www.notion.so/chanrankim/2c2967ff842344a78a04f50d8274c802?pvs=4)에 맞춰서 1~73 라인의 title, discription, 그리고 제목을 고쳐주시길 바랍니다! 

Long short term memory(LSTM) 같은 경우 본문에서는 LSTM을 쓰겠지만, 제목에서는 장단기 기억(LSTM) 으로 위키백과 등 백과사전 등재 용어로 한번만 한국어 사용 부탁드립니다..!

movie5 reviewed (changes requested):
스타일가이드라인에 맞춰서 1~73라인 번역 부탁드려요!

Jieun-Park99 committed:
✏️ comment : gan / dcgan / index.html
✏️ comment : lstm / index.html
✏️ comment : normalization / layer_norm / index.html
✏️ comment : optimizers / adam_warmup_cosine_decay.html

Jieun-Park99 commented:
코멘트 수정하여 commit 완료하였습니다!
